develocity-gradle-plugin-adapters/build.gradle.kts

```diff
@@ -32,3 +32,9 @@ develocityAdaptersPublication {
     name = "Develocity Gradle API adapters for Develocity and Gradle Enterprise Gradle plugins"
     description = "Adapter interfaces for interacting with the Develocity and Gradle Enterprise Gradle plugins"
 }
+
+tasks.withType<Jar>().configureEach {
```

Comment:
Can you do: 

```kotlin
tasks.withType<Jar>().configureEach {
    into(".") {
        from(rootProject.layout.projectDirectory.file("LICENSE"))
    }
}
```

This is more symmetric with the other repositories. I forget what implications accessing `rootProject.layout` has for isolated projects, but I thought that was a safe API because it is a read only object. It might be worth checking.

Reply:
Changed